0.Preprocess_Data/pre-processing.sh

```diff
@@ -15,11 +16,11 @@ cell_types=( PBMC SHSY5Y )
 aggragates=( True False )
 nomics=( True False )
 
-for cell_type in $cell_types; do
+for cell_type in ${cell_types[@]}; do
```

Comment:
Just curious, what does this `@` symbol do here? Is this related to papermill?

Reply:
No, these specify to loop through the entire array